packages/kit/test/apps/options-2/test/test.js

```diff
@@ -7,7 +7,7 @@ test.describe.configure({ mode: 'parallel' });
 
 test.describe('paths.base', () => {
 	test('serves /basepath', async ({ page }) => {
-		await page.goto('/basepath');
+		await page.goto('/basepath/');
```

Comment:
what's the reason for this change?

Reply:
Vite only serves the index page with a trailing slash

Reply:
Related issue: https://github.com/vitejs/vite/issues/8770

Reply:
okay, but the path is incorrect